.github/workflows/tests_primary.yml

```diff
@@ -66,3 +66,34 @@ jobs:
     - run: npm run build
     - run: node lib/cli/cli install --with-deps
     - run: npm run ttest
+
+  test_docker_mode:
+    name: Docker Mode ${{ matrix.os }} (${{ matrix.browser }})
```

Comment:
Seems like you deleted the wrong GitHub Action workflow file.

Reply:
reverted. The original idea was to cut on our CI usage, but turns out we have to test both `root` and `pwuser` usecases.